MANIFEST.in

```diff
@@ -1,7 +1,3 @@
-include LICENSE
-
-include etc/requirements*.txt
-
-include tests/resources *.csv
-include versioneer.py
 include exchange_calendars/_version.py
```

Comment:
setuptools_scm writes this file even if it is not listed in manifest. Line is not needed I suppose

Reply:
I was under the impression setuptools_scm only includes files that are being tracked? (the PR adds `_version.py` to gitignore to avoid registering changes on installing an editable version.)

Reply:
I have checked that if `_version` file is not at project at all, setuptools_scm still writes it during installation

Reply:
Thanks! Removed in the 'Update dependencies' commit.